tools/get_data_ibm_store.py

```diff
@@ -0,0 +1,111 @@
+import urllib
+
```

Comment:
consider adding a comment here about how this is a terribad use of discovery, and what running the script actually produces?